sda-commons-server-auth-testing/src/main/java/org/sdase/commons/server/opa/testing/OpaClassExtension.java

```diff
@@ -2,17 +2,17 @@
 
 import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
 
+import com.github.tomakehurst.wiremock.junit5.WireMockExtension;
 import com.github.tomakehurst.wiremock.matching.RequestPatternBuilder;
 import org.junit.jupiter.api.extension.AfterAllCallback;
 import org.junit.jupiter.api.extension.BeforeAllCallback;
 import org.junit.jupiter.api.extension.ExtensionContext;
-import org.sdase.commons.client.jersey.wiremock.testing.WireMockExtension;
 
 @SuppressWarnings("WeakerAccess")
 public class OpaClassExtension extends AbstractOpa implements BeforeAllCallback, AfterAllCallback {
 
   private final WireMockExtension wireMockExtension =
-      new WireMockExtension(wireMockConfig().dynamicPort());
+      new WireMockExtension.Builder().options(wireMockConfig().dynamicPort()).build();
```

Comment:
Not for this PR, but in another extension (like this), we might want to use `WireMockServer` directly.